util/session/sessionmanager.go

```diff
@@ -563,6 +570,16 @@ func (mgr *SessionManager) VerifyToken(tokenString string) (jwt.Claims, string,
 			return nil, "", common.ErrTokenVerification
 		}
 
+		id, _ := claims["jti"].(string)
```

Comment:
better to check if the conversion to string was successful using an `ok` boolean (instead of ignoring). If the key is not found or if the conversion fails, id can be set to empty string probably with a warning log.